app/views/conference/schedule.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <% if @conference.program.events.scheduled.any? %>
+  <% if @rooms.length > 1 and @conference.program.events.scheduled.any? %>
```

Comment:
@Ana06  Is there any reason why its > 1 and not >= 1 ?
Right now if there is only 1 room, the schedule won't be displayed, even if there are events scheduled in that one room.